### PR TITLE
Add EffectRandomF<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: functions/increment.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#ifndef FUNCTIONS_INCREMENT_F_H
```
Comment (profezzorn) [inline functions/increment.h:1]: This doesn't belong in this PR.

**Comment by fdarosa2663** [comment]
must have crossed up branches, will resubmit
